### ISSUE TITLE
New install for vscode - can't create account to login

### ISSUE THREAD

**Issue by HarveySummers** [kiteco/vscode-plugin#156]
Installed kite for vscode/win10, prompted to create account - says kite unavailable, then goes back to prompt to create account. 

Could this be a blocked port or proxy issue? Can't find any documentation addressing this type of problem

Please advise

**Comment by abe33** [comment]
Hello @HarveySummers, thanks for the report. Do you have any restrictions in place? Can you try using the `Kite: Install` command that should be available in the command palette and tell me if it's any better?

**Comment by HarveySummers** [comment]
So Kite seems to be quasi-loaded. VSCode has a purple bar that says kite:not logged in after I click on the reload button. Login says it doesn't have an email for me, so I try to create an account - that takes me to the kite download page.  typing "kite:install" does nothing, yet I see that some of the kite commands are in the user setup because they are showing some errors. I have uninstalled/reinstalled and always get stuck at this point.

![image](https://user-images.githubusercontent.com/24304778/44007326-417ea268-9e61-11e8-8c87-f46bdcd6e5ed.png)

**Comment by abe33** [comment]
Thanks for the reply, so as far as I can tell, the install suceeded (because kite is saying that you're not logged and that a file is not whitelisted, so kite is running on your computer), that's one thing out of the way. As for creating an account, we currently don't have a form in VSCode to create an account, only to log into kite (arguably we should either provide account creation as well or just drop log in to avoid confusion). You should be able to create an account using the Kite copilot app (available either by clicking on the kite icon in the tray or by using the `Kite: Open Copilot` command in VSCode). Hope it helps.

**Comment by HarveySummers** [comment]
The kite commands don’t show up and kite isn’t listed as an extension.  When I reinstall, I get the welcome and can’t login or create an account. I can’t run any kite command.

[cid:image002.jpg@01D432FF.57C16B50]

**Comment by ghstahl** [comment]
I just installed as well and it is stuck on create account in the Kite Install.

Failure from both my home and at work on different machines.
VS Code 1.26.1 

connect ETIMEDOUT 137.116.115.94:80

<my email>@gmail.com
 Continue

**Comment by garrett-vangilder** [comment]
I'm also receiving this same error. I installed Kite, and cannot get past the account creation page.

**Comment by abe33** [comment]
Hi there, and sorry for the late reply @ghstahl and @garrett-vangilder, is there any useful message besides the ETIMEDOUT error? Are you able to create an account when downloading and installing Kite from kite.com? I never had this issue in my tests but I'll give another look.

**Comment by garrett-vangilder** [comment]
@abe33 the only message I received was the Timedout error. I actually uninstalled the plugin from vscode, and will redownload now. 

Okay, I have an update. So I was able to successfully install your platform. There are 2 different views that are opened in VS Code. The first is the kite install view, and the second is the Kite Status view. To install go to the Kite Status view, and click install. This will open the web browser and then allow the user to install the necessary application. The bug that myself and @ghstahl were referring to is found on the Kite Install view in VS Code. I pressed the install button, and still received the timeout error. 

So there is a workaround but it is very confusing for new users.
